content/docs/quickstart.mdx

```diff
@@ -22,74 +22,198 @@ keywords:
+
+```bash
+ARCH=[your arch]
+OS=[your os]
+VERSION=[desired version]
+curl -L https://github.com/pomerium/cli/releases/download/${VERSION}/pomerium-cli-${OS}-${ARCH}.tar.gz \
+    | tar -z -x
+```
+
+</TabItem>
+<TabItem value="homebrew" label="Homebrew">
+
+```bash
+brew tap pomerium/tap
+brew install pomerium-cli
+```
+
+</TabItem>
+<TabItem value="source" label="Source">
+
+```bash
+git clone git@github.com:pomerium/cli.git
+cd pomerium
+make build
+./bin/pomerium-cli --help
+```
+
+</TabItem>
+
+</Tabs>
 
 :::
 
-## Next Steps
+### Connect to Verify
+
+In the Zero Console, select the **From** route for the Verify app.
 
-If you want to [try Enterprise](https://www.pomerium.com/enterprise-sales/), check out the [Enterprise with Docker quickstart](/docs/deploy/enterprise/quickstart).
+For example:
 
-If you want to try connecting Pomerium with other services, see some of our [Guides](/docs/guides).
+![Selecting the external URL for the Verify route in the Zero Console](./img/quickstart/pz-quickstart-verify-route.png)
 
-**Did you finish this quickstart guide?** We'd love to hear what you think. Get in touch with us on our [Discuss forum](https://discuss.pomerium.com/), message us on [Twitter](https://twitter.com/pomerium_io), [LinkedIn](https://www.linkedin.com/company/pomerium-inc), or check out our [Community](https://www.pomerium.com/docs/community) page.
+:::note
 
-:::caution
+Because the Verify service uses a publically accessible URL, Pomerium Zero won't be able to connect to the `jwks.json` endpoint to fetch the signing key used to verify the user's JWT.
 
-This is a test environment! If you followed all the steps in this doc your Pomerium environment is not using trusted certificates. Remember to use a valid certificate solution before moving this configuration to a production environment. See [Certificates](/docs/concepts/certificates) for more information.
+![Identity verification failed error when accessing the Verify service](./img/quickstart/pz-identity-verification-failed.png)
 
 :::
 
-[configuration file]: /docs/reference
-[container images]: https://hub.docker.com/r/pomerium/pomerium
-[docker]: https://docs.docker.com/install/
-[docker compose]: https://docs.docker.com/compose/install/
+
+## Build your own policy
+
+In the **Policy** tab, create a new policy that instructs Pomerim to grant access only if the user's email address ends the specified value.
+
+![Building a basic policy in Pomerium Zero](./img/quickstart/pz-my-policy.png)
+
+### Attach the policy to a route
+
+Attach your new policy to the Verify route:
+
+![Attach the new policy to the Verify route in the Zero Console](./img/quickstart/pz-attach-policy.png)
+
+Now, if you authenticate using an email address with a domain other than the one specified in the policy, Pomerium Zero will deny the user access.
+
+## Next Steps
+
+Now that you've completed the Pomerium Zero Quickstart, build your first route:
+
+- **Pomerium Zero Fundamentals: Build Routes**
```

Comment:
This doesn't currently link to anything. Are we waiting on the fundamentals to be built too?

Reply:
Yes, we are.